pyodide_build/pywasmcross.py

```diff
@@ -597,7 +605,7 @@ def handle_command(
         if tmp is None:
             # No source file, it's a query for information about the compiler. Pretend we're
             # gfortran by letting gfortran handle it
-            return subprocess.run(line).returncode
+            return subprocess.run(line, check=False).returncode
```

Comment:
```suggestion
            return subprocess.run(line, check=True).returncode
```

Reply:
Again should be false, we return the return code. With `check=True` it will throw unless the returncode is 0 so in context `check=True` makes no sense.